redis/src/types.rs

```diff
@@ -121,9 +120,23 @@ pub enum Value {
     /// to express nested structures.
     Bulk(Vec<Value>),
     /// A status response.
-    Status(String),
+    Status(String), // maybe change to SimpleString ?
     /// A status response which represents the string "OK".
     Okay,
+    /// Ordered map value from the server. use `as_map_iter` function.
+    Map(Vec<Value>),
+    /// Unordered set value from the server.
+    Set(Vec<Value>),
+    /// A floating number response from the server.
+    Double(f64),
+    /// A boolean response from the server.
+    Boolean(bool),
+    /// First String is format and other is the string
+    VerbatimString(String, String),
```

Comment:
instead of the first string, maybe use an enum, since there are only 2 legal values - text and markdown.

Reply:
Consider adding field names, for clarity.

Reply:
I also have added Unknown just in case